Modelica/Electrical/Analog/Examples/OpAmps/package.order

```diff
@@ -12,7 +12,7 @@ VoltageFollower
 Comparator
 InvertingSchmittTrigger
 SchmittTrigger
+LCOscillator
```

Comment:
Any reason for the changed order?

Reply:
Yes I tried to order the examples according to the problems they did rise during the last regression tests ;-)
I see no big harm in changing the order ...